fiware-region-sanity-tests/tests/fiware_region_with_networks_tests.py

```diff
@@ -347,116 +480,37 @@ def test_deploy_instance_with_network_and_associate_public_ip(self):
         # Associate Public IP to Server
         self.nova_operations.add_floating_ip_to_instance(server_id=server_id, ip_address=allocated_ip)
 
-    def test_deploy_instance_with_networks_and_e2e_connection_using_public_ip(self):
+    def test_deploy_instance_with_new_network_and_e2e_connection_using_public_ip(self):
         """
-        Test whether it is possible to deploy and instance, assign an allocated public IP and establish a SSH connection
+        Test whether it is possible to deploy an instance with new network, assign an allocated public IP
+        and establish a SSH connection
```

Comment:
You are writing the description of the test using two lines but the `nosetests` lib does not process the second one (only the first one). Then, the _tooltip_ on the Dashboard's Test Detail page will not show the whole description...